splitter.py

```diff
@@ -184,6 +185,70 @@ def validate_filenames(directory, repoinfo):
     return isok
 
 
+def _get_recursive_dependencies(all_deps, idx, stream):
+    if stream.get_NSVCA() in all_deps:
+        # We've already encountered this NSVCA, so don't go through it again
+        logging.debug('Already included {}'.format(stream.get_NSVCA()))
+        return
+
+    # Store this NSVCA/NS pair
+    all_deps.add(stream.get_NSVCA())
+
+    logging.debug("Recursive deps: {}".format(stream.get_NSVCA()))
+
+    # Loop through the dependencies for this stream
+    deps = stream.get_dependencies()
+
+    # At least one of the dependency array entries must exist in the repo
+    found_dep = False
+    for dep in deps:
+        # Within an array entry, all of the modules must be present in the index
+        found_all_modules = True
+        for modname in dep.get_runtime_modules():
+            # Ignore "platform" because it's special
+            if modname == "platform":
+                logging.debug('Skipping platform')
+                continue
```

Comment:
Is `platform` the only case we want to ignore?

Reply:
`platform` is the only special-case for runtime. If that ever changes, we'll need to modify this (but we'd also need to modify it in many other projects/libraries)